examples/next/pages/ui/components/in-app-messaging/demo/utils.ts

```diff
@@ -2,10 +2,10 @@ import { useState, useEffect } from 'react';
 import * as Analytics from '@aws-amplify/analytics';
 import { syncMessages } from 'aws-amplify/in-app-messaging';
 import {
-  Message,
-  MessageAction,
-  MessageButton,
-  MessageLayout,
+  InAppMessage,
+  InAppMessageAction,
+  InAppMessageButton,
+  InAppMessageLayout,
 } from '@aws-amplify/ui-react-core-notifications';
```

Comment:
This import should be from '@aws-amplify/ui-react-notifications' if we are exposing the types?